### ISSUE TITLE
Update Tutorial with New Instructions

### ISSUE THREAD

**Issue by fabiogomezdiaz** [ibm-cloud-architecture/refarch-cloudnative-kubernetes#46]
https://cloudcontent.mybluemix.net/devops/method/tutorials/microservices-app-on-kubernetes

**Comment by ileneruth** [comment]
Hey Fabio, Does this include new instructions for installing dependencies?  The scripts referenced in task #2 (eg. ./install_cli.sh) don't seem to exist in the repo.

**Comment by jkwong888** [comment]
I think it will, and it will likely just point at existing documentation on how to install kubectl and helm.  we removed that script because it was extra maintenance burden and not much value added.

**Comment by fabiogomezdiaz** [comment]
@ileneruth +1 to what @jkwong888 said

**Comment by ileneruth** [comment]
Makes sense not to maintain that script, but pointers are really needed.  The one I personally wasn't familiar with was jq.

Also, the tutorial is really problematic right now, as you go through task one and then it advises you to verify success by looking for: install_bluecompute_ce.sh which doesn't seem to be there.  Proceed to Task 2 and additional referenced scripts aren't there.  At that point, I don't think most people would carry on.

**Comment by jkwong888** [comment]
hence the task to update the tutorial 😄  .. maintaining different versions of a script for windows, mac, and linux users that could break anytime when the upstream utilities change is a lot more work than updating the tutorial once and for all, just have to find a pocket of time to do it.

BTW i don't think `jq` is needed anymore.  If you read the README.md on the github repo, it contains all of the information you need, we will be translating that to the tutorial once we have a moment to breathe.

**Comment by MelGHub** [comment]
@fabiogomezdiaz  , I need to deploy the bleucompute-ce on IBM's POWER (ppc64le) worker node of ICP 3.1. I see that this repo contains the helm chart, however could not locate the Dockerfiles. Assuming that the images that gets pulled as part of deployment are not ppc64le enabled, do you think having the Dockefiles will allow us to build them? anything else that you can think of might be a blocker to Deploy the bluecompute on ppc64le?

**Comment by fabiogomezdiaz** [comment]
@MelGHub the docker files are located in the root folder of all the repos listed here:
https://github.com/ibm-cloud-architecture/refarch-cloudnative-kubernetes/tree/spring#project-repositories

Please look at the first 7 repos (minus the first one), switch to the `spring` branch, and you will find that all have a Dockerfile in them.

As far as `ppc64le` I would recommend doing the following:
- Start by creating a `Dockerfile.ppc64le` for each repo
- Build the docker files and run them locally
- Look into each of the repos to find instructions on how to test each microservice
- If you are confident your testing works, then push those docker images to a docker hub registry (or to a private registry if you prefer)
- Then try to deploy the helm charts with those new images
  + i.e. you could deploy the `inventory` microservice as follows:
  ```bash
  helm upgrade --install inventory --set service.type=NodePort \
    --set mysql.existingSecret=inventory-mysql \
    --set image.repository=YOUR_REGISTRY/bluecompute-inventory-ppc64le \
    chart/inventory
  ```
- Then you can test again to make sure everything works

Technically there should not be a need to change the helm chart itself outside of just providing the image as it should "just work" assuming you are using the right docker image and there is not pp64le specific settings you should put in the chart.

Outside of that, for the databases (MySQL, MariaDB, ElasticSearch, and CouchDB) we are using the community provided charts, which we don't support ourselves. So you would have to find `ppc64le` versions for those as well or create your own. Here is the source for those charts:
- https://github.com/helm/charts/tree/master/stable/mysql
- https://github.com/helm/charts/tree/master/stable/mariadb
- https://github.com/helm/charts/tree/master/incubator/elasticsearch
- https://github.com/helm/charts/tree/master/incubator/elasticsearch

The charts' `values.yaml` will have the docker image that each chart is using and you can track it from there in DockerHub or whatever registry they are using and find whether they have a `ppc64le` counter part.

Hope this helps
